WickedEngine/wiBacklog.h

```diff
@@ -66,6 +66,12 @@ namespace wi::backlog
 	LogLevel GetUnseenLogLevelMax();
 
 	void SetLogFile(const std::string& path);
+	void GetLogFile(std::string& path);
```

Comment:
Why not return a std::string instead? Having a getter filling a parameter feel weird to me.